lib/domUpdates.js

```diff
@@ -1,19 +1,37 @@
 const domUpdates = {
 
-  renderScoreboard(players) {
-    players.forEach((player) => {
+  renderScoreboard (players) {
+    players.forEach(player => {
       let ul = document.querySelector('.scoreboard');
-      let li = document.createElement('li');
-      ul.appendChild(li)
-      li.innerHTML = `Player: ${player.name}
-                      <br>
-                      Score: ${player.score}`;
+      let div = document.createElement('div');
+      ul.appendChild(div)
+      div.innerHTML = `<li>Player: ${player.name}</li>
+                      <li>Score: ${player.score}</li>`;
+    })
```

Comment:
This HTML structure is a bit odd -- usually you won't see a `div` inside of a `ul` I'd refactor this to just append a single `li` tag per player - if you want to separate out the styling for the player name and player score, you can wrap the score value in a `span` tag within the `li`. e.g.:

```js
<li>Player: ${player.name} <span class="score">Score: ${player.score}</span></li>`